app/controllers/group_sets_controller.rb

```diff
@@ -11,7 +11,11 @@ def load_resource
       when :index
         @group_sets = GroupSet.all.desc(:created_at).page(params[:page]).per(10)
       when :show
-        @group_set = GroupSet.find(params[:id])
+        if request.fullpath == "/"
+          @group_set = GroupSet.last
```

Comment:
`last`でもいいのですが、L12の書き方に合わせて以下の書き方の方がわかりやすいかもですね:eyes:
```suggestion
          @group_set = GroupSet.all.desc(:created_at).first
```

Reply:
@narukami894 意図するデータ取得としては確かにこちらが適切でした。細かくとも重要な部分ご指摘ありがとうございます！